modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -141,11 +141,17 @@ function createStandaloneFromProvider({
         mapboxApiAccessToken: mapboxApiKey,
         onLoad: modifyMapboxElements
       });
+    case 'carto':
+      deck.log.info('Using Carto base maps')();
+      return new deck.DeckGL({
+        ...sharedProps,
+        ...props
+      });
```

Comment:
Does this mean the carto API key is never used? Or is it just not explicit like `googleMapsKey` below?

Reply:
It's just not explicit, at least at the moment. I'll focus on getting the carto layers integrated in a separate PR.